NOpenType/ToPixelRasterizer.cs

```diff
@@ -2,34 +2,37 @@
 
 namespace NRasterizer
 {
-    public class ToPixelRasterizer: IGlyphRasterizer
+    /// <summary>
+    /// Converts from glyph point based values into pixel based ones
+    /// </summary>
+    /// <remarks>
+    /// This is a struct because its super short lived and it prevents the need for an allocation for each character rendered.
+    /// This is internal as it will only ever be used by the <see cref="Renderer"/> and should not be used by external systems.
+    /// </remarks>
+    internal struct ToPixelRasterizer
```

Comment:
Yeah, that's definitely not right. This should only be created once per DrawText, or even less. You don't have to fix that problem in this pr of course, but lets keep it a class for now. You can drop a TODO remarking this class is instantiated for each glyph which is not right.